### ISSUE TITLE
i have a probelm like when im clicking at the bottom of my screen i can accidentally open an app in my taskbar

### ISSUE THREAD

**Issue by roblosplaya** [mengdeveloper/microsoftrbx-cursor-fix#31]
i have a probelm like when im clicking at the bottom of my screen i can accidentally open an app in my taskbar

**Comment by zzahkaboom24** [comment]
Can you show me a small video? it might be because I changed the mouse locking behavior around the window. If that's the case, I will revert that change

**Comment by roblosplaya** [comment]
https://github.com/mengdeveloper/microsoftrbx-cursor-fix/assets/143820926/bff34680-b3fb-43c5-97e9-03c80cbf4997

**Comment by roblosplaya** [comment]
whenever i play pvp games are smth and requires clicking i some times accidentally open apps on my taskbar

**Comment by zzahkaboom24** [comment]
Yea, I am on it already. Thanks

**Comment by roblosplaya** [comment]
yw

**Comment by zzahkaboom24** [comment]
@roblosplaya 
Check if this Release fixes your issue
[x64.CursorFix.zip](https://github.com/mengdeveloper/microsoftrbx-cursor-fix/files/12504330/x64.CursorFix.zip)

**Comment by roblosplaya** [comment]
so uh
it now happens with the window.

**Comment by roblosplaya** [comment]
i mean i wont use my mouse on the top of my screen is it alr

**Comment by roblosplaya** [comment]
wait found out smth it doesnt work in first person and third person

**Comment by zzahkaboom24** [comment]
@roblosplaya Can you show me another video of what you mean? in my testing it works just fine

**Comment by roblosplaya** [comment]
ima try again

**Comment by roblosplaya** [comment]
alr so it works on 1 and 2 but not 3 and after i enable i cant disable

**Comment by roblosplaya** [comment]
@zzahkaboom24 it only works on 1 and 2 i believe

**Comment by zzahkaboom24** [comment]
@roblosplaya After disabling, press alt+tab once to switch the focus to another window, it will disable it fully after

**Comment by zzahkaboom24** [comment]
@roblosplaya You said 3 does not work, but 1 & 2 works as expected, right?
What with 3 exactly does not work, or can you show me a video where it does not work?

**Comment by roblosplaya** [comment]
alr

**Comment by roblosplaya** [comment]
alr nvm it works

**Comment by roblosplaya** [comment]
at first 3 was making the cursor still go to taskbar

**Comment by zzahkaboom24** [comment]
@roblosplaya Yes, I will give you another release cuz I now changed the values to be a bit more inside the window, I will send it to you in 2 Minutes

**Comment by zzahkaboom24** [comment]
@roblosplaya Try this release out to see if this fixes your cursor going to the taskbar on the bottom and to the title bar on the top
The adjusted values should theoretically fix this issue and make everything work as you expect now.
[x64.CursorFix.V1.0.7.zip](https://github.com/mengdeveloper/microsoftrbx-cursor-fix/files/12504422/x64.CursorFix.V1.0.7.zip)
